index.html

```diff
@@ -5,19 +5,34 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Document</title>
+    <title>Poonicorn</title>
 </head>
 <body>
-    <section>
-        <button><img src="./assets/rainbow.jpeg"></button>
-        <button></button>
-        <button></button>
-        <button></button>
-        <button></button>
-        <button></button>
-        <button></button>
-    </section>
-
-<script type="module" src="./src/app.js"></script>    
+    <div>
+        <header>
+            <h1 class="rainbow">Taste the rainbow with me!</h1>
+        </header>
+    
+            <section>
+                <button id="red" class="color" value="red"><img class="button-image" src="./assets/rainbow.jpeg" alt="cute rainbow picture"></button>
+                <button id="orange" class="color" value="orange"><img class="button-image" src="./assets/rainbow.jpeg" alt="cute rainbow picture"></button>
+                <button id="yellow" class="color" value="yellow"><img class="button-image" src="./assets/rainbow.jpeg" alt="cute rainbow picture"></button>
+                <button id="green" class="color" value="green"><img class="button-image" src="./assets/rainbow.jpeg" alt="cute rainbow picture"></button>
+                <button id="blue" class="color" value="blue"><img class="button-image" src="./assets/rainbow.jpeg" alt="cute rainbow picture"></button>
+                <button id="purple" class="color" value="purple"><img class="button-image" src="./assets/rainbow.jpeg" alt="cute rainbow picture"></button>
+                <button id="pink" class="color" value="pink"><img class="button-image" src="./assets/rainbow.jpeg" alt="cute rainbow picture"></button>
+            </section>
+        
+            <section id="poonicorn">
+                <img id="unicorn" src="./assets/unicornBody.jpeg" alt="pooping unicorn">
+                <!--below is our intial static body(poo)-->
```

Comment:
This looks good! In the future instead of commenting out dead code, we want PR's to production branches to be clean so just delete it instead!